Modix.Bot/Modules/ReplModule.cs

```diff
@@ -175,26 +175,33 @@ private async Task<EmbedBuilder> BuildEmbedAsync(SocketGuildUser guildUser, Resu
 
             embed.AddField(a => a.WithName("Code").WithValue(Format.Code(parsedResult.Code, "cs")));
 
+            // in the following code, 'size' is - 2 (minus 'cs') and then added with however long the embed type is
```

Comment:
Isn't this backwards?

e.g. json
`1014 - 2 + 4 = 1016`, but it should actually be `1012`, since "json" leaves less room for the field text than "cs" does.

Reply:
i'm a horrible person